src/renderer/lib/beat_util.ts

```
@@ -47,3 +47,24 @@
+  console.log(beat);
+  if (beat.image?.type) {
+    if (beat.image.type === "movie") {
+      return false;
+    }
+    if (beat.image.type === "beat") {
+      return false;
+    }
+  }
+  if (beat.htmlPrompt) {
+    return false;
+  }
```

ESLint

[error] Replace this if-then-else flow by a single return statement.

(sonarjs/prefer-single-boolean-return)